### PR TITLE
Rename handle vision position estimate

### PR DESCRIPTION
**Pull request by Ryanf55**
The function takes position (linear) + orientation (angular), therefore it's a pose, not a position.

Reference: https://robotics.stackexchange.com/questions/19785/whats-the-difference-between-pose-measurement-and-position-measurement

### CODE REVIEW

File: libraries/GCS_MAVLink/GCS_Common.cpp [hunk 1]
```diff
@@ -3995,7 +3995,7 @@ void GCS_MAVLINK::handle_common_message(const mavlink_message_t &msg)
         break;
 
     case MAVLINK_MSG_ID_VISION_POSITION_ESTIMATE:
-        handle_vision_position_estimate(msg);
+        handle_vision_pose_estimate(msg);
```
Comment (Ryanf55) [inline libraries/GCS_MAVLink/GCS_Common.cpp:3998]: Leave it as is.

File: libraries/GCS_MAVLink/GCS.h [hunk 2]
```diff
@@ -1005,7 +1005,7 @@ class GCS_MAVLINK
     virtual void handle_manual_control_axes(const mavlink_manual_control_t &packet, const uint32_t tnow) {};
 
     void handle_vicon_position_estimate(const mavlink_message_t &msg);
-    void handle_vision_position_estimate(const mavlink_message_t &msg);
+    void handle_vision_pose_estimate(const mavlink_message_t &msg);
```
Comment (rmackay9) [inline libraries/GCS_MAVLink/GCS.h:1008]: Let's leave this one as-is because it matches the mavlink message being consumed.

Comment (Ryanf55) [reply libraries/GCS_MAVLink/GCS.h]: Done

File: libraries/GCS_MAVLink/GCS_Common.cpp (hunk 1 above)
Comment (Ryanf55) [reply libraries/GCS_MAVLink/GCS_Common.cpp]: Done

**Comment by Ryanf55** [comment]
Ready for review again, I took out the name `vision` since it's redundant with the class name.

**Review by rmackay9 — APPROVED** [review]

**Pull request merged**